annotations/views/repository_views.py

```diff
@@ -87,26 +88,29 @@ def repository_collections(request, repository_id):
     return render(request, "annotations/repository_collections.html", context)
 
 
-# Since in citesphere_authenticated we already have a login_required decorator, we don't need to add another one here
 @citesphere_authenticated
 def repository_collection(request, repository_id, group_id):
-    """View to fetch and display collections within Citesphere Groups"""
+    """View to fetch and display collections and group texts within Citesphere Groups"""
     params = _get_params(request)
 
     repository = get_object_or_404(Repository, pk=repository_id)
     
     manager = RepositoryManager(user=request.user, repository=repository)
+
+    page = int(request.GET.get('page', 1))
     
     try:
         response_data = manager.collections(groupId=group_id)
         group_info = response_data.get('group')
         collections = response_data.get('collections', [])
+        group_texts =  manager.group_items(groupId=group_id, page=page)
     except IOError:
         return render(request, 'annotations/repository_ioerror.html', {}, status=500)
 
     project_id = request.GET.get('project_id')
     
-    base_url = reverse('repository_collection', args=(repository_id, group_id))
+    items_per_page = getattr(settings, 'PAGINATION_PAGE_SIZE')
```

Comment:
Why don't you just use `settings. PAGINATION_PAGE_SIZE` and use it directly in line 113?

Reply:
I changed it to `settings.PAGINATION_PAGE_SIZE`